packages/backend/src/modules/activity2/Activity2Module.ts

```diff
@@ -36,9 +39,8 @@ export function createActivity2Module(
     logger = logger.for('Activity2Module')
     logger.info('Starting')
```

Comment:
are those logs needed?

Reply:
yes, that's the pattern we're following for all modules